specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/containerInstance.json

```diff
@@ -0,0 +1,1560 @@
+          "description": "The name of the volume mount.",
+          "type": "string"
+        },
+        "mountPath": {
+          "description": "The path within the container where the volume should be mounted. Must not contain colon (:).",
+          "type": "string"
+        },
+        "readOnly": {
+          "description": "The flag indicating whether the volume mount is read-only.",
+          "type": "boolean"
+        }
+      }
+    },
+    "ContainerExec": {
+      "type": "object",
+      "description": "The container execution command, for liveness or readiness probe",
+      "properties": {
+        "command": {
+          "description": "The commands to execute within the container.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ContainerHttpGet": {
+      "type": "object",
+      "description": "The container Http Get settings, for liveness or readiness probe",
+      "properties": {
+        "path": {
+          "description": "The path to probe.",
+          "type": "string"
+        },
+        "port": {
+          "type": "integer",
+          "description": "The port number to probe.",
+          "format": "int32"
+        },
+        "scheme": {
+          "description": "The scheme.",
+          "type": "string",
+          "enum": [
+            "http",
+            "https"
+          ]
+        }
+      },
+      "required": [
+        "port"
+      ]
+    },
+    "ContainerProbe": {
+      "type": "object",
+      "description": "The container probe, for liveness or readiness",
+      "properties": {
+        "exec": {
+          "description": "The execution command to probe",
+          "$ref": "#/definitions/ContainerExec"
+        },
+        "httpGet": {
+          "description": "The Http Get settings to probe",
+          "$ref": "#/definitions/ContainerHttpGet"
+        },
+        "initialDelaySeconds": {
+          "description": "The initial delay seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "periodSeconds": {
+          "description": "The period seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "failureThreshold": {
+          "description": "The failure threshold.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "successThreshold": {
+          "description": "The success threshold.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeoutSeconds": {
+          "description": "The timeout seconds.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "ContainerGroup": {
+      "description": "A container group.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "type": "object",
+          "required": [
+            "properties"
+          ],
+          "identity": {
```

Comment:
This is in the wrong place and is the cause for the autorest codegen to fail.  Presumably this is supposed to go under `properties`?